pyproject.toml

```diff
@@ -0,0 +1,89 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "edumfa"
+version = "2.2.0"
+requires-python = ">=3.8"
+authors = [{ name = "edumfa.io", email = "edumfa@listserv.dfn.de" }]
+description = "eduMFA: identity, multifactor authentication (OTP), authorization, audit"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+license = { file = "LICENSE" }
+keywords = ["OTP", "two factor authentication", "management", "security"]
+classifiers = [
+    "Framework :: Flask",
+    "License :: OSI Approved :: GNU Affero General Public License v3",
+    "Programming Language :: Python",
+    "Development Status :: 5 - Production/Stable",
+    "Topic :: Internet",
+    "Topic :: Security",
+    "Topic :: System :: Systems Administration :: Authentication/Directory",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12"
+]
+dynamic = ["dependencies"]
+
+[project.urls]
+Homepage = "https://edumfa.io/"
+Documentation = "https://edumfa.readthedocs.io/en/v2.1.0/"
```

Comment:
Maybe we can change that to a more generic link so we don't have to update it everytime?